svc-task/thandle/thandle.go

```diff
@@ -158,6 +158,19 @@ func (ts *TasksRPC) DeleteTask(ctx context.Context, req *taskproto.GetTaskReques
 	if err != nil {
 		return nil
 	}
+	privileges := []string{common.PrivilegeConfigureManager}
+	authStatusCode, authStatusMessage := ts.AuthenticationRPC(req.SessionToken, privileges)
```

Comment:
the definition of the auth function has been changed. please make changes according to it